app/presenters/sufia/work_show_presenter.rb

```diff
@@ -24,5 +32,16 @@ def itemtype
     rescue
       'http://schema.org/CreativeWork'
     end
+
+    private
+
+      def featured?
+        @featured ||= !FeaturedWork.find_by_generic_work_id(solr_document.id).nil?
```

Comment:
This isn't going to memoize if `@featured` is `false`.  You have to explicitly check to see if `@featured.nil?`  Also, why not `FeaturedWork.where(generic_work_id: solr_document.id).exists?` That should avoid the overhead of instantiation.